roles/openshift_manage_node/tasks/main.yml

```diff
@@ -19,10 +19,12 @@
     -n default
   register: omd_get_node
   until: omd_get_node.rc == 0
-  retries: 50
+  retries: 10
   delay: 5
   changed_when: false
   with_items: "{{ openshift_nodes }}"
+  when: hostvars[item].openshift.node.nodename is defined
+  ignore_errors: true
```

Comment:
Agreed. There is no `any_errors_fatal: true` so I'm not yet sure why this is causing the whole play to error out, rather than continue with nodes that succeeded.